api/api_ethereum.go

```diff
@@ -334,13 +370,28 @@ func (api *EthereumAPI) GetProof(ctx context.Context, address common.Address, st
 // * When blockNr is -1 the chain head is returned.
 // * When blockNr is -2 the pending chain head is returned.
 func (api *EthereumAPI) GetHeaderByNumber(ctx context.Context, number rpc.BlockNumber) (map[string]interface{}, error) {
-	// TODO-Klaytn: Not implemented yet.
-	return nil, nil
+	klaytnHeader, err := api.publicBlockChainAPI.GetHeaderByNumber(ctx, number)
+	// In Ethereum, err is always nil because the backend of Ethereum always return nil.
+	err = nil
```

Comment:
It could be looked weird, but currently Ethereum backend always return nil at error position.

[Ethereum source code link](https://github.com/ethereum/go-ethereum/blob/356bbe343a30789e77bb38f25983c8f2f2bfbb47/eth/api_backend.go#L65-L76).